pkgs/development/libraries/onnxruntime/default.nix

```diff
@@ -171,7 +182,8 @@ effectiveStdenv.mkDerivation rec {
     "-DFETCHCONTENT_QUIET=OFF"
     "-DFETCHCONTENT_SOURCE_DIR_ABSEIL_CPP=${abseil-cpp.src}"
     "-DFETCHCONTENT_SOURCE_DIR_DATE=${howard-hinnant-date}"
-    "-DFETCHCONTENT_SOURCE_DIR_FLATBUFFERS=${flatbuffers}"
+    "-DFETCHCONTENT_SOURCE_DIR_FLATBUFFERS=${flatbuffers.src}"
+    "-DFETCHCONTENT_SOURCE_DIR_GOOGLETEST=${gtest.src}"
```

Comment:
i feel like `flatbuffers` should fail given issues like https://github.com/microsoft/onnxruntime/issues/14981; maybe the local build i tested out was flawed and it's actually not gonna work in CI